nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -22,7 +22,8 @@
                                            plot_roi, plot_stat_map, plot_epi,
                                            plot_glass_brain, plot_connectome,
                                            plot_connectome_strength,
-                                           plot_prob_atlas, plot_carpet,
+                                           plot_markers, plot_prob_atlas, 
```

Comment:
We'll still need the tests for `plot_connectome_strength` since it's still included in the code base